apps/site/pages/docs/customizing/components.mdx

```diff
@@ -163,6 +163,97 @@ To ensure visual consistency with your store's theme, we highly recommend utiliz
 
 ---
 
+## Importing FastStore UI component styles
+
+If you are using a [FastStore UI component](/components) and want to apply its styles to your 
+FastStore project, follow the steps below. In this example, we will override the 
+[`ProductTitle`]() component within the [`ProductDetails`]() section. 
+
+### Before you begin
+As the [Customizing a new component when using override](/docs/customizing/components#customizing-a-new-component-when-using-override)
+you should also have a basic understanding of [overriding native components and props](/docs/building-sections/overriding-components-and-props).
```

Comment:
Not sure about it, maybe it just for a new section, but for this example make sense.